weave/trace/weave_client.py

```diff
@@ -408,7 +412,9 @@ def __init__(
         self.project = project
         self.server = server
         self._anonymous_ops: dict[str, Op] = {}
+        self.async_job_queue = AsyncJobQueue()
         self.ensure_project_exists = ensure_project_exists
+        atexit.register(self._cleanup)
```

Comment:
should we put this on a dunder method? post_init? @andrewtruong would know where

Reply:
Not sure what you are recommending here. We do call cleanup in __del__ as well.